app/views/govuk_publishing_components/components/docs/add_another.yml

```diff
@@ -83,3 +83,37 @@ examples:
           <label for="employee_1_name" class="gem-c-label govuk-label">Full name</label>
           <input class="gem-c-input govuk-input" id="employee_1_name" name="employee[1]name">
         </div>
+
+  with_select_with_search:
+    description: Add another with a select with search input
+    data:
+      fieldset_legend: "Organisation"
+      add_button_text: "Add another organisation"
+      items:
+        - fields: >
+            <div class="govuk-form-group">
+            <select name="organisations[0][id]" id="organisations_1" class="govuk-select" data-module="select-with-search" tabindex="-1">
```

Comment:
If you could add a label to this select then we won't get an accessibility failure warnings in the Component Guide I think.